.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Test build
-      run: go build -o fakemachine cmd/fakemachine/main.go
+      run: go build -o fakemachine -ldflags="-X main.Version=$(git describe --tags HEAD)" cmd/fakemachine/main.go
```

Comment:
I'd rather build the version as a variable (which could be overwritten?) first in a separate step and output as variable.

Reply:
This should be fixed now